clang/test/CIR/CodeGen/atomic-runtime.cpp

```diff
@@ -16,23 +16,23 @@ int runtime_load(int *ptr, int order) {
 
 // CHECK: %[[ptr:.*]] = cir.load %[[ptr_var:.*]] : !cir.ptr<!cir.ptr<!s32i>>, !cir.ptr<!s32i>
 // CHECK: %[[order:.*]] = cir.load %[[order_var:.*]] : !cir.ptr<!s32i>, !s32i
-// CHECK: cir.switch (%[[order]] : !s32i) [
-// CHECK: case (default) {
+// CHECK: cir.switch (%[[order]] : !s32i) {
+// CHECK: cir.case(default, []) {
 // CHECK:   %[[T8:.*]] = cir.load atomic(relaxed) %[[ptr]] : !cir.ptr<!s32i>, !s32i
 // CHECK:   cir.store %[[T8]], %[[temp_var:.*]] : !s32i, !cir.ptr<!s32i>
 // CHECK:   cir.break
-// CHECK: },
-// CHECK: case (anyof, [1, 2] : !s32i) {
+// CHECK: }
+// CHECK: cir.case(anyof, [#cir.int<1> : !s32i, #cir.int<2> : !s32i]) {
```

Comment:
I wish there was an easy way to not repeat the type for each element!

Reply:
Maybe we can make it with manual printer and manual parser? If we can, we can make it in following patches. I don't feel it is too bad since CIR may not be wrote manually. Ideally, CIR will be read and wrote by the program most of the time.

Reply:
I feel the same, we could have a custom printer/parser just for the list part. It would be nice to have but definitely not a priority.